### DISCUSSION TITLE
Support inlay hints.

### DISCUSSION THREAD

**Discussion by byehack** [microsoft/pylance-release#2643] [Enhancement]
inlay hints can be usefull for:
show parameters name of callable objects and functions.
better showing errors/warnings.
showing type hints.

**Comment by jerryc05** [comment]
I totally agree. Inlay hints from the JS/TS plugins are awesome and Python should also deserve this cool feature with the type inferred by Pylance.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Comment by luabud** [comment]
Inlay hints are available now: https://devblogs.microsoft.com/python/python-in-visual-studio-code-july-2022-release/#inlay-type-hints
